corehq/sql_db/routers.py

```diff
@@ -43,6 +44,8 @@ def allow_migrate(db, app_label):
         )
     elif app_label == SQL_ACCESSORS_APP:
         return db in partition_config.get_form_processing_dbs()
+    elif app_label == WAREHOUSE_APP:
```

Comment:
yeah not necessarily a bad thing though. this is just used in the partition router, and i don't think we ever want the warehouse db to be in the main db. can update